types/html-webpack-tags-plugin/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for html-webpack-tags-plugin 2.0
+// Project: https://github.com/jharris4/html-webpack-tags-plugin
+// Definitions by: // Definitions by: 贺师俊 <https://github.com/hax>, Vladimir Grenaderov <https://github.com/VladimirGrenaderov>, Max Boguslavskiy <https://github.com/maxbogus>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
+
+import { Plugin } from 'webpack';
+
+export interface Options {
+    tags: string | string[];
+    files?: string | string[];
+    append?: boolean;
+}
+
+export default class HtmlWebpackTagsPlugin extends Plugin {
```

Comment:
According to the source, this is exported with `module.exports =` not `exports.default =`, so this import would not work at runtime unless additional compiler options are specified. I would recommend using the `export =` syntax to emulate the actual behavior, and move `Options` into an `HtmlWebpackTagsPlugin` namespace that merges with the class definition:

```ts
class HtmlWebpackTagsPlugin extends Plugin {
  constructor(options: HtmlWebpackTagsPlugin.Options);
}
namespace HtmlWebpackTagsPlugin {
  interface Options { ... }
}
export = HtmlWebpackTagsPlugin;
```